services/cache.js

```diff
@@ -114,93 +59,77 @@ cache.init = async () => {
   // Create the redis instance.
   cache.client = redis.createClient();
 
-  // Load the INCR_SCRIPT and DECR_SCRIPT into Redis.
-  let [incrScriptHash, decrScriptHash] = await Promise.all([
-    loadScript('INCR_SCRIPT', INCR_SCRIPT),
-    loadScript('DECR_SCRIPT', DECR_SCRIPT)
-  ]);
+  // This is designed to increment a key and add an expiry iff the key already
```

Comment:
_iff_ I was a boy 🎶

Reply:
I use _iff_ as shorthand for _if and only if_ 😄